.github/workflows/ci.yml

```diff
@@ -88,10 +88,30 @@ jobs:
           cd vcpkg
           ./bootstrap-vcpkg.sh -disableMetrics
 
+      - name: Configure HealthGPS (debug)
```

Comment:
Might ask to go over this with you in future. Starting to get quite big, and I'm not totally comfortable with the actions files yet.

Reply:
Sure... We could chat tomorrow if you like?

Reply:
Sure, if you have a moment. Thanks